src/components/IconBadge.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
-import styled from '@stream-io/styled-components';
+import { View } from 'react-native';
 
-const Container = styled.View``;
+import { styled } from '../styles/styledComponents';
```

Comment:
is this not how all of the other `styled-component` imports should be?

Reply:
So the plan is to shift to using the typed version as we move files to TS...any place referencing theme will show red ;)